From be82ecea0cc4fe45627a709d3ddeb14fc8030b20 Mon Sep 17 00:00:00 2001
From: Sebastien Gonzalve <sebastien.gonzalve@intel.com>
Date: Mon, 5 Nov 2012 09:06:05 +0100
Subject: GTI: Add Gti in services list.

BZ: 69831
[CLEAN]

GTI stands for the Global test interface, it is used to probe debugging/audit
information from AUD with the help of phonetool.

Category: device enablement
Domain: AudioComms-Merrifield
Origin: internal
Upstream-Candidate: no, proprietary
Change-Id: I53c1fcfd0885653a5891e578b4a4f3e83bb05abb
Orig-Change-Id: I21b71661b3d51c38c436ce454320d29f1e22f4ea
Signed-off-by: Selma Bensaid <selma.bensaid@intel.com>
Signed-off-by: Sebastien Gonzalve <sebastien.gonzalve@intel.com>
---
 cmds/servicemanager/service_manager.c |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/cmds/servicemanager/service_manager.c b/cmds/servicemanager/service_manager.c
index 3eaf1eb..a918c19 100644
--- a/cmds/servicemanager/service_manager.c
+++ b/cmds/servicemanager/service_manager.c
@@ -32,6 +32,7 @@ static struct {
     { AID_MEDIA, "media.player" },
     { AID_MEDIA, "media.camera" },
     { AID_MEDIA, "media.audio_policy" },
+    { AID_MEDIA, "media.gti_service"},
     { AID_DRM,   "drm.drmManager" },
     { AID_NFC,   "nfc" },
     { AID_BLUETOOTH, "bluetooth" },
-- 
1.7.4.1

